src/server/responses.py

```diff
@@ -0,0 +1,115 @@
+from flask import Response, make_response
+from jwt import InvalidTokenError
+
+from .database import Message
+
+
+def me(uuid: str, email: str, wallet_usd: float, wallet_btc: float) -> Response:
```

Comment:
Can all of these function be grouped into a class enum? I.e.:
```python
class RESPONSES(Callable, Enum):
    ME: Callable = me_function
    ME2: Callable = lambda uuid, email, wallet_usd, wallet_btc: ...
```

And use either function objects or lambdas in there? This would prevent importing whole `responses.py` file in `main.py`.